{{cookiecutter.project_slug}}/src/features/{% if cookiecutter.has_maps_blueprint == "True" %}MapsScreen{% endif %}/index.js

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from "react";
+import { StyleSheet, Image, View, Alert, } from "react-native";
+import { Text, Layout, Button } from "react-native-ui-kitten";
+import LogoIcon from "../../assets/images/backgroundLoginV1.png";
+import { installed_blueprints } from "../../config/installed_blueprints";
+import { store } from "../../redux/store";
+import MapView, { PROVIDER_GOOGLE } from 'react-native-maps';
+import { SearchBar } from 'react-native-elements';
+
+
+export default class MapsScreen extends Component {
+  componentDidMount() {
+    store.dispatch({ type: "TEST/ALO" });
+  }
+
+  state = {
+    search: '',
+  };
+
+  updateSearch = search => {
+    this.setState({ search });
+  };
+
+  render() {
+    const { search } = this.state;
+
+    return [
+      <MapView
+        provider={PROVIDER_GOOGLE} // remove if not using Google Maps
```

Comment:
Since we are installing Google Maps I think we always want to have the provider right?